cypress/e2e/tests/migration/applicationinventory/assessment/copy_assessment/copy_assessment.test.ts

```diff
@@ -132,12 +133,14 @@ describe(["@tier2"], "Copy assessment and review tests", () => {
         cy.get(copy).should("be.visible").should("not.be.disabled");
 
         // Select all applications
+        cy.log("Selecting all...");
```

Comment:
Do we need this ? All logs consume memory.

Reply:
Did that for debug, I will remove it

Reply:
Done